spotbugs-include.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+    <Match>
+        <Bug category="I18N" />
+    </Match>
+</FindBugsFilter>
```

Comment:
Do you think this should be added as a separate PR or is fine to include it in this one even though it seems not directly related to the parsing issue--maybe it is and I am wrong?

Reply:
This is helpful to find instances where the code is relying on the default encoding. See https://spotbugs.readthedocs.io/en/latest/bugDescriptions.html?highlight=i18n#dm-reliance-on-default-encoding-dm-default-encoding

Reply:
I did not add more rules because we are in violation of _many_ different rules, but I didn't see any types that would create platform issues.

If you'd like to see the full list, delete line 3 and run the gradle task, you'll see hundreds of things!